micro_dl/light/engine.py

```diff
@@ -44,9 +45,11 @@ def __init__(
         self.loss_function = loss_function if loss_function else F.mse_loss
         self.lr = lr
         self.schedule = schedule
+        self.log_num_samples = log_num_samples
         self.training_step_outputs = []
         self.validation_step_outputs = []
-        self.example_input_array = torch.rand(1, 1, 5, 256, 256)  # required to log the graph
+        # required to log the graph
+        self.example_input_array = torch.rand(1, 1, 5, 256, 256)
```

Comment:
@ziw-liu this line leads to the following output on terminal. I added this so that lightning can log the graph.
  | Name  | Type    | Params | In sizes            | Out sizes          
------------------------------------------------------------------------------
0 | model | Unet25d | 2.0 M  | [1, 1, 5, 256, 256] | [1, 1, 1, 256, 256]
------------------------------------------------------------------------------

This should be `config[data][yx_patchsize]`.
I can't figure out how to access the config parameters inside LightningModule at runtime. Can you look into this?

Reply:
See https://github.com/mehta-lab/microDL/pull/232/commits/ca56416453620a14d5dd94ce40f1a11860761bb5